src/tests/eventpipe/GCEvents.cs

```diff
@@ -188,8 +190,15 @@ public async void GCCollect_ProducesVerboseEvents()
                 Action _eventGeneratingAction = () => 
                 {
                     List<string> testList = new List<string>();
-                    for(int i = 0; i < 100000000; i ++)
+                    for (int i = 0; i < 100_000_000; i ++)
                     {
+                        if (RuntimeInformation.ProcessArchitecture == Architecture.X86)
```

Comment:
Why is there a need for GC collect in these last few tests? And why only on x86 here? Sounds pretty weird, and at least needs a comment. Feels like a brittle test with this.

Reply:
I should have commented better but without the GC collect, no unload module events get sent on x86 only and the test fails because it expects at least one unload event.

Reply:
I got this mixed up with another test but on x86 we weren't getting all the GC events that the test required unless I did the GC collect.